src/containers/about-page.js

```diff
@@ -20,7 +20,6 @@ export default connect(
 )(AboutPage);
 
 function mapStateToProps() {
-  return {};
 }
```

Comment:
this line got replaced with this:
return { albumLinks: Selectors.getAlbumLinks(state) };